test/non_ascii_connection_strings_test.dart

```diff
@@ -0,0 +1,208 @@
+import 'dart:io';
+
+import 'package:docker_process/containers/postgres.dart';
+import 'package:path/path.dart' as p;
+import 'package:postgres/postgres.dart';
+import 'package:test/expect.dart';
+import 'package:test/scaffolding.dart';
+
+void main() {
+  /// This cannot be run on CI as each test requires different pg_hba.conf
+  /// to be mounted to the container. 
+  if (Platform.environment.containsKey('GITHUB_ACTION')) {
```

Comment:
I haven't really checked yet, but IIRC Github Action is able to run `docker run` and other commands... have you tried it?

Reply:
Yeah I believe it's possible but not with the current setup (e.g. using `services.postgres`)... I'll give it another shot later and try a different setup.

Reply:
Note: a repository can have multiple workflows, and we can differentiate tests either via directory or tags.

Reply:
(Also: thank you for looking into this!)

Reply:
Yeah that what I was thinking -- to tag tests with a `solo` tag and exclude them from common tests and then run all `solo` tests with `concurrency=1`.